dtfs-central-api/api-tests/v1/portal-facility-amendments/amendment-get.api-test.ts

```diff
@@ -0,0 +1,135 @@
+  });
+
+  afterAll(() => {
+    process.env = originalEnv;
+  });
+
+  describe('with FF_PORTAL_FACILITY_AMENDMENTS_ENABLED set to `false`', () => {
+    beforeAll(() => {
+      process.env.FF_PORTAL_FACILITY_AMENDMENTS_ENABLED = 'false';
+    });
+
+    it('returns 404 when the amendment does not exist', async () => {
+      const { status } = await testApi.get<FacilityAmendmentResponse>(generateUrl(facilityId, amendmentId));
+
+      expect(status).toEqual(HttpStatusCode.NotFound);
+    });
+  });
+
+  describe('GET /v1/portal/facilities/:facilityId/amendments/:amendmentId', () => {
+    describe('with FF_PORTAL_FACILITY_AMENDMENTS_ENABLED set to `true`', () => {
+      beforeAll(() => {
+        process.env.FF_PORTAL_FACILITY_AMENDMENTS_ENABLED = 'true';
+      });
+
+      it('returns 400 when the facility id is invalid', async () => {
+        const anInvalidFacilityId = 'InvalidId';
+
+        const { status, body } = await testApi.get<FacilityAmendmentResponse>(generateUrl(anInvalidFacilityId, amendmentId));
+
+        expect(status).toEqual(HttpStatusCode.BadRequest);
+
+        expect(body).toEqual({
+          message: "Expected path parameter 'facilityId' to be a valid mongo id",
+          code: API_ERROR_CODE.INVALID_MONGO_ID_PATH_PARAMETER,
+        });
+      });
+
+      it('returns 400 when the amendment id is invalid', async () => {
+        const anInvalidAmendmentId = 'InvalidId';
+
+        const { status, body } = await testApi.get<FacilityAmendmentResponse>(generateUrl(facilityId, anInvalidAmendmentId));
+
+        expect(status).toEqual(HttpStatusCode.BadRequest);
+
+        expect(body).toEqual({
+          message: "Expected path parameter 'amendmentId' to be a valid mongo id",
+          code: API_ERROR_CODE.INVALID_MONGO_ID_PATH_PARAMETER,
+        });
+      });
+
+      it('returns 404 when the facility does not exist', async () => {
+        const aValidButNonExistentFacilityId = new ObjectId().toString();
+
+        const { status, body } = await testApi.get<FacilityAmendmentResponse>(generateUrl(aValidButNonExistentFacilityId, amendmentId));
+
+        expect(status).toEqual(HttpStatusCode.NotFound);
+        expect(body).toEqual({
+          status: HttpStatusCode.NotFound,
+          message: `Facility not found: ${aValidButNonExistentFacilityId}`,
+        });
+      });
+
+      it('returns 404 when the amendment does not exist', async () => {
+        const aValidButNonExistentAmendmentId = new ObjectId().toString();
+
+        const { status, body } = await testApi.get<FacilityAmendmentResponse>(generateUrl(facilityId, aValidButNonExistentAmendmentId));
+
+        expect(status).toEqual(HttpStatusCode.NotFound);
+        expect(body).toEqual({
+          status: HttpStatusCode.NotFound,
+          message: `Amendment not found: ${aValidButNonExistentAmendmentId}`,
+        });
+      });
+
+      // TODO: DTFS2-7674 - add success tests after making a PUT request
```

Comment:
I only realised when it came to writing these api tests that I cant check the endpoints working without another endpoint to create an entry. I think its best to get this merged in & in the PUT PR I'll update this test (since I need the types from this PR I want it merged in first)